src/libraries/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/DefaultValueInstruction.cs

```diff
@@ -21,6 +22,8 @@ internal DefaultValueInstruction(Type type)
 
         public override string InstructionName => "DefaultValue";
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2077:UnrecognizedReflectionPattern",
+            Justification = "_type is a ValueType. You can always create an instance of a ValueType.")]
```

Comment:
Note: this usage of `Activator.CreateInstance` and the ones below in LocalAccess.cs should be changed to `GetUninitializedObject` with #45397.